js/app/service/timezoneservice.js

```diff
@@ -48,7 +48,7 @@ app.service('TimezoneService', ['$rootScope', '$http', 'Timezone', 'TimezoneList
 
 			_this._timezones[tzid] = $http({
 				method: 'GET',
-				url: $rootScope.baseUrl + 'timezones/' + tzid + '.ics'
+				url: OC.generateUrl('/apps/calendar/v1/timezones/') + tzid + '.ics'
```

Comment:
this won't work when calendar is in a different app folder
Let's fix `$rootScope.baseUrl` instead ;)